stable/contour/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+name: contour
+version: 0.2.0
+description: Contour envoy based ingress controller installation
+appVersion: "v0.9.0"
```

Comment:
v0.11.0 is out. Should this be updated? I do realize this is the result of the PR taking awhile.